README.md

```diff
@@ -71,6 +71,10 @@ The arguments are the following:
         files, including the unchanged ones, will be created in the workspace.
         (default: CHANGED_ONLY)
 
+  [--maxFixesPerRule <maxFixesPerRule>]
+        Mode for outputting files: Max number of fixes per rule.
+        (default: Integer.MAX_VALUE )
+
```

Comment:
This doesn't match what is actually going to be shown to the user. To get the correct description, the best way is to build the jar and to call it by passing "-h".

Reply:
done